x-pack/legacy/plugins/infra/server/routes/inventory_metadata/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Boom from 'boom';
+import { boomify } from 'boom';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+import { InfraBackendLibs } from '../../lib/infra_types';
+import { throwErrors } from '../../../common/runtime_types';
+import { getAWSMetadata } from './lib/get_aws_metadata';
+import {
+  InventoryMetaRequestRT,
+  InventoryMetaWrappedRequest,
+  InventoryMetaResponse,
+  InventoryMetaResponseRT,
+} from '../../../common/http_api/inventory_meta_api';
+
+export const initInventoryMetaRoute = (libs: InfraBackendLibs) => {
+  const { framework } = libs;
+
+  framework.registerRoute<InventoryMetaWrappedRequest, Promise<InventoryMetaResponse>>({
```

Comment:
I think the format for registering routes has changed. I'm getting a 404 when I try to load the page and all the other routes now have the NP style routes.

Reply:
The format has changed, it should be the only NP change that really affects things. Let me know if I can help.